week3/Week 3 Playground.playground/Contents.swift

```diff
@@ -0,0 +1,123 @@
+import SwiftUI
+
+// a)
+let range = 0...20
+var nums: [Int] = []
+
+for num in range {
+  nums.append(num)
+}
+for num in nums where num % 2 == 0{
+  print(num)
+}
+print("------")
+
+// b)
```

Comment:
alterntive 

```swift
let sentence = "The qUIck bRown fOx jumpEd over the lAzy doG"

let vowels = "aeiou"
var vowelCount = 0

for char in sentence.lowercased() {
    if vowels.contains(char) {
        vowelCount += 1
    }
}

print("The number of vowels in the sentence is:", vowelCount)

```

Reply:
I forgot about `.lowercased()`, that is a cleaner way to do it for sure.